nmigen/back/verilog.py

```diff
@@ -65,6 +66,12 @@ def _convert_rtlil_text(rtlil_text, *, strip_internal_attrs=False, write_verilog
     if popen.returncode:
         raise YosysError(error.strip())
     else:
+        # Remove empty lines and things that are not legal Verilog
+        # syntax at start of file
```

Comment:
Instead of explaining what the code below does (which is fairly clear from the code itself), this comment should be explaining why this code is necessary, similar to what I asked for in the other PR.